src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -89,15 +105,20 @@ public void postToldpcvOfLdprsWithoutMementoDatetimeMustIgnoreBody(final String
                                         "Memento-Datetime header MUST ignore any request body.",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-post",
                                         ps);
-
+        // create the versioned resource
+        final String newResource = createVersionedResource(uri, info).body().asString();
+        // create a memento for the current time
+        final Response versionResponse = doPost(newResource + "/fcr:versions", "Send some random data here");
+        confirmPresenceOfLinkValue(MEMENTO_LINK_HEADER, versionResponse);
+        confirmPresenceOfVersionLocationHeader(versionResponse);
```

Comment:
Per the requirement being tested, I would expect this test to also be verifying that the state of the created memento does not include the `POST`ed body.